subprojects/android-test/test-app/src/androidTest/kotlin/com/avito/android/ui/test/PageObjectTest.kt

```diff
@@ -39,12 +39,20 @@ class PageObjectTest {
     }
 
     @Test
-    fun element__notFound__byDefaultMatcherInTheWrongParent() {
+    fun parent__assertionError__byDefaultMatcherInTheWrongParent() {
         rule.launchActivity(PageObjectActivity.intent(R.layout.page_object_2)) // twin
 
-        exception.expect(NoMatchingViewException::class.java)
+        exception.expect(junit.framework.AssertionFailedError::class.java)
```

Comment:
It was hard to realize how it worked before. And why now it fails with `junit.framework.AssertionFailedError::class.java`. @eugene-krivobokov  Could you check the correctness of that fix?